package.json

```diff
@@ -35,10 +35,13 @@
 		"@types/node": "^12.0.10",
 		"@types/sinon": "^9.0.9",
 		"chai": "^4.2.0",
+		"esm": "^3.2.25",
 		"form-data": "^2.3.3",
+		"fs": "0.0.1-security",
```

Comment:
should be removed as it's not a real package https://www.npmjs.com/package/fs

Reply:
done